cms/envs/aws_migrate.py

```diff
@@ -1,30 +0,0 @@
-"""
```

Comment:
@omarkhan @bradenmacdonald @jibsheet The configuration repo still references aws_migrate.py.  Is there a corresponding change there that is yet to be merged?